src/agent/src/linux_abi.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2019 Ant Financial
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Linux ABI related constants.
+
+pub const SYSFS_DIR: &str = "/sys";
+
+pub const SYSFS_PCI_BUS_PREFIX: &str = "/sys/bus/pci/devices";
```

Comment:
It's really annoying that constant string cannot be joined at compile time in rust, but adding https://github.com/Vurich/const-concat seems overkill here.